_layouts/sidenav.html

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
I like the idea of having this layout 👍  I can plan to follow-up to update the "Contact us" page to use this as well.